src/test/java/org/sagebionetworks/web/unitclient/presenter/SettingsPresenterTest.java

```diff
@@ -293,6 +293,13 @@ public void testAddEmailNewEmail(){
 	}
 	
 	@Test
+	public void testAddEmailNewEmailWithSpaces(){
+		String email3 = " testuser3@test.com ";
+		presenter.addEmail(email3);
+		verify(mockSynapseClient).additionalEmailValidation(anyString(), anyString(), anyString(), any(AsyncCallback.class));

```

Comment:
Please verify that trimmed email 3 is passed to the synapse client (use Mockito eq() instead of anyString())